### PR TITLE
testing/fractal: new aport

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Cogitri** [comment]
Needs #7241

**Comment by TBK** [comment]
@Cogitri Could I get you to trigger a CI rerun? (close/open PR should do the trick).

**Comment by Cogitri** [comment]
Right, thanks for the reminder.

File: testing/fractal/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--buildtype=release \
+		. output
+	ninja -C output
+}
+
+check() {
```
Comment (TBK) [inline testing/fractal/APKBUILD]: There does not seem to be a test suite:
```
ninja: Entering directory `output'
[0/1] Running all tests.
No tests defined.
>>> fractal: Entering fakeroot...
ninja: Entering directory `output'
[0/1] Installing files. 
```

**Comment by Cogitri** [comment]
Pushed new version

**Pull request closed without merging**